composer.json

```diff
@@ -21,14 +21,14 @@
         "ext-curl":"*",
         "phpcr/phpcr": "~2.1.2",
         "phpcr/phpcr-utils": "1.*,>=1.1.2",
-        "jackalope/jackalope": "~1.2.0"
+        "jackalope/jackalope": "dev-node-get-definition as 1.2.1"
```

Comment:
update once https://github.com/jackalope/jackalope/pull/292 is merged and we were able to tag a release